src/outbreak_data/outbreak_data.py

```diff
@@ -276,47 +263,45 @@ def sequence_counts(location=None, cumulative=None, sub_admin=None, server=serve
         df = pd.DataFrame(raw_data['results'])
     return df
 
-def mutations_by_lineage(mutation, location=None, pango_lin=None, datemin=None, datemax=None, freq=None, server=server):
+def mutations_by_lineage(mutation=None, location=None, pango_lin=None, datemin=None,  datemax=None, freq=None, server=server):
     """Returns the prevalence of a mutation or series of mutations across specified lineages by location
 
     Arguments:
-     :param mutations: (Optional). List of mutations. 
-     :param location_id: (Optional). If not specified, return most recent date globally.
+     :param mutations: (Optional). List or string of mutations separated by ",". 
+     :param location_id: (Optional). A string; If not specified, return most recent date globally.
      :param pangolin_lineage: (Optional). If not specfied, returns all Pango lineages containing that mutation.
      :param frequency: (Optional) Minimimum frequency threshold for the prevalence of a mutation in a lineage.
      :param datemin: (Optional). A string representing the first cutoff date for returned date. Must be in YYYY-MM-DD format and be before 'datemax'
      :param datemax: (Optional). A string representing the second cutoff date. Must be in YYY-MM-DD format and be after 'datemin'
      :return: A pandas dataframe.
     """
-
-    if isinstance(mutation, list):
-        pass
-    elif isinstance(mutation, str):
-         mutation = mutation.replace(" ", "")
-         mutation = list(mutation.split(","))
     
-    mutations = '' + ','.join(mutation) + ''   
-    
-    query = f'mutations={mutations}'
+    if mutation: 
+        if isinstance(mutation, str):
+             mutation = mutation.replace(" ", "")
+             mutation = list(mutation.split(","))
+        mutation = '' + ' AND '.join(mutation) + ''   
+        query = f'mutations={mutation}'
     
+    if pango_lin and '*' in pango_lin:
```

Comment:
Right now, if `pango_lin` doesn't contain a star, it doesn't get included in the query, and if `pango_lin` is included in the query, then the mutations aren't. In general, users might want to specify both

Reply:
At the moment, it looks like there's also an API bug where the results aren't quite right when specifying both a lineage and a mutation, but Everaldo's fixing that now and otherwise this code looks correct to me regarding that